build_common.ps1

```diff
@@ -940,19 +735,17 @@ class BuildProject {
 		# The CookPackages commandlet generally is super unhelpful. The output is basically always the same and errors
 		# don't occur -- it rather just crashes the game. Hence, we just pipe the output to $null
 		Write-Host "Invoking CookPackages (this may take a while)"
-		$cook_args = @("-platform=pcconsole", "-quickanddirty", "-modcook", "-sha", "-multilanguagecook=INT+FRA+ITA+DEU+RUS+POL+KOR+ESN", "-singlethread", "-nopause")
+		$cook_args = @("cookpackages", "-platform=pcconsole", "-quickanddirty", "-modcook", "-sha", "-multilanguagecook=INT+FRA+ITA+DEU+RUS+POL+KOR+ESN", "-singlethread", "-nopause")
 		if ($this.final_release -eq $true)
 		{
 			$cook_args += "-final_release"
 		}
 		
-		# TODO: Look for the crash message
-		& "$($this.commandletHostPath)" CookPackages @cook_args >$null 2>&1
-
-		if ($LASTEXITCODE -ne 0)
-		{
-			ThrowFailure "Failed to cook native script packages!"
-		}
+		# TODO: Filter more lines for HL cook? `Hashing`? `SHA: package not found`? `Couldn't find localized resource`?
+		# `Warning, Texture file cache waste exceeds`? `Warning, Package _ is not conformed`?
+		$handler = [BufferingReceiver]::new()
```

Comment:
Is the `TODO` still relevant with the introduction of the `BufferingReceiver`?

Reply:
In combination with figuring out what can commonly cause the HL cooker to crash I guess?